shpc/settings.yml

```
@@ -34,6 +34,13 @@ default_version: true
 # It's recommended to do this for faster loading
 container_base: $root_dir/containers
 
+# If defined, create a simplified "symlink install" that shortens module names
+# to be just the container, e.g., ghcr.io/autamus/samtools -> module load samtools
+symlink_base: null
```

Could there not be a default value like for the modules and the containers ? esp. as the generation of symlinks is anyway controlled by the boolean `symlink_tree`
```suggestion
symlink_base: $root_dir/symlinks
```

---

I like that idea - what say you @marcodelapierre ?

---

Tested with this commit https://github.com/muffato/singularity-hpc/commit/5b26c910cc85a8cdb8b7f695d0a8caf865801c51 , which also updates `.gitignore`

---

added with https://github.com/singularityhub/singularity-hpc/pull/502/commits/10cf953d9c29b282781b0e50c10bd4b3ab7295ef thank you!

---

Still unable to trial the new branch, just reading through this PR.

If what you are saying is to have a non null default for `symlink_base`, yes, I totally agree, because the feature is controlled by the logical `symlink_tree`, not by the value of `symlink_base`

---

yes 100%! And it's been added.